exporter/redis.go

```diff
@@ -81,13 +81,20 @@ var (
 
 		// # Persistence
 		"rdb_changes_since_last_save":  "rdb_changes_since_last_save",
+		"rdb_bgsave_in_progress":       "rdb_bgsave_in_progress",
+		"rdb_last_save_time":           "rdb_last_save_time",
+		"rdb_last_bgsave_status":       "rdb_last_bgsave_status",
 		"rdb_last_bgsave_time_sec":     "rdb_last_bgsave_duration_sec",
 		"rdb_current_bgsave_time_sec":  "rdb_current_bgsave_duration_sec",
+		"rdb_last_cow_size":            "rdb_last_cow_size",
```

Comment:
Is this in bytes? 
In that case, should this be `rdb_last_cow_size_bytes` ?

Reply:
Yes, this in bytes:

https://redis.io/commands/info
```
rdb_last_cow_size: The size in bytes of copy-on-write allocations during the last RBD save operation
```

I renamed metric.